src/screens/AirdropDashboard/components/AirdropInProgress.tsx

```diff
@@ -105,12 +82,91 @@ export const AirdropInProgress: FC<Props> = props => {
         </Text>
       </View>
       <Countdown dataEnd={CONST.airdropDate} />
-      <AirdropInProgressContent {...props} />
+      {loading && (
+        <View style={styles.loadingContainer}>
+          <Loading />
+        </View>
+      )}
+      {error && (
+        <View style={styles.errorContainer}>
+          <Error />
+        </View>
+      )}
+      {!error && !loading && (
```

Comment:
it's hard to read this code, would be nice to split it to components

Reply:
done